doc/classes/OptionButton.xml

```diff
@@ -74,6 +74,15 @@
 				Return the ID of the item at index "idx".
 			</description>
 		</method>
+                <method name="get_item_index" qualifiers="const">
+                        <return type="int">
+                        </return>
+                        <argument index="0" name="id" type="int">
+                        </argument>
+                        <description>
+                                Return the index of the item with id "id".
```

Comment:
Use `[code]id[/code]` for the argument name. And you can avoid repeating "id" with e.g. "with the given [code]id[/code]".